packages/frontend-design-poc/src/components/Header/index.ts

```diff
@@ -0,0 +1 @@
+export { Header } from './Header.tsx'
```

Comment:
Could this be `export * from './Header.tsx';`? that way we don't need to change the file if we export an extra component from the file.